src/Kovan-DssSpell.sol

```diff
@@ -28,45 +38,255 @@ contract SpellAction {
+        OsmAbstract(PIP_UNI).kiss(MCD_END);
+        // Set UNI Osm in the OsmMom for new ilk
+        // !!!!!!!! Only if PIP_UNI = Osm
+        OsmMomAbstract(OSM_MOM).setOsm(ILK_UNI_A, PIP_UNI);
+
+        // Set the UNI-A debt ceiling
+        VatAbstract(MCD_VAT).file(ILK_UNI_A, "line", 15 * MILLION * RAD);
+        // Set the UNI-A dust
+        VatAbstract(MCD_VAT).file(ILK_UNI_A, "dust", 100 * RAD);
+        // Set the Lot size
+        CatAbstract(MCD_CAT).file(ILK_UNI_A, "dunk", 500 * RAD);
+        // Set the UNI-A liquidation penalty (e.g. 13% => X = 113)
+        CatAbstract(MCD_CAT).file(ILK_UNI_A, "chop", 113 * WAD / 100);
+        // Set the UNI-A stability fee (e.g. 1% = 1000000000315522921573372069)
+        JugAbstract(MCD_JUG).file(ILK_UNI_A, "duty", THREE_PERCENT_RATE);
+        // Set the UNI-A percentage between bids (e.g. 3% => X = 103)
+        FlipAbstract(MCD_FLIP_UNI_A).file("beg", 103 * WAD / 100);
+        // Set the UNI-A time max time between bids
+        FlipAbstract(MCD_FLIP_UNI_A).file("ttl", 1 hours);
+        // Set the UNI-A max auction duration to
+        FlipAbstract(MCD_FLIP_UNI_A).file("tau", 1 hours);
+        // Set the UNI-A min collateralization ratio (e.g. 150% => X = 150)
+        SpotAbstract(MCD_SPOT).file(ILK_UNI_A, "mat", 175 * RAY / 100);
+
+        // Update UNI-A spot value in Vat
+        SpotAbstract(MCD_SPOT).poke(ILK_UNI_A);
+
+        // Add new ilk to the IlkRegistry
+        IlkRegistryAbstract(ILK_REGISTRY).add(MCD_JOIN_UNI_A);
+
+        // Set gulp amount in faucet on kovan (only use WAD for decimals = 18)
+        FaucetAbstract(FAUCET).setAmt(UNI, 2500 * WAD);
+
+        // Update the changelog
+        CHANGELOG.setAddress("UNI", UNI);
+        CHANGELOG.setAddress("MCD_JOIN_UNI_A", MCD_JOIN_UNI_A);
+        CHANGELOG.setAddress("MCD_FLIP_UNI_A", MCD_FLIP_UNI_A);
+        CHANGELOG.setAddress("PIP_UNI", PIP_UNI);
+
+        //
+        // Add renBTC
+        //
 
-        // Set new Chief in the Pause
-        DSPauseAbstract(MCD_PAUSE).setAuthority(MCD_ADM);
+        // Add RENBTC-A ilk
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).vat() == MCD_VAT, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).ilk() == RENBTC_ILK, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).gem() == RENBTC, "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).dec() == DSTokenAbstract(RENBTC).decimals(), "join-dec-not-match");
+        require(FlipAbstract(MCD_FLIP_RENBTC_A).vat() == MCD_VAT, "flip-vat-not-match");
+        require(FlipAbstract(MCD_FLIP_RENBTC_A).cat() == MCD_CAT, "flip-cat-not-match");
+        require(FlipAbstract(MCD_FLIP_RENBTC_A).ilk() == RENBTC_ILK, "flip-ilk-not-match");
 
-        // Set new Chief in the FlipperMom
-        FlipperMomAbstract(FLIPPER_MOM).setAuthority(MCD_ADM);
+        SpotAbstract(MCD_SPOT).file(RENBTC_ILK, "pip", PIP_RENBTC);
```

Comment:
nit: I would change `RENBTC_ILK` to `RENBTC_A_ILK` variable for clarity

Reply:
changing to `ILK_RENBTC_A` to match `UNI` above.

Reply:
done